framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1869,7 +1869,7 @@ protected void checkAndPerformFlowAnalysis(Tree tree) {
     if (analysis.isRunning()) {
       as = analysis.getValue(tree);
     }
-    if (as == null) {
+    if (as == null && flowResult != null) {
```

Comment:
There seems to be a reoccurring problem where the Value Checker is called during parsing and `flowResult` is null. The `if (stubTypes.isParsing()...` code a few lines above isn't sufficient to prevent this from happening.  So, I just added a check for null.  (This should probably be in a separate pull request?)